rust/operator-binary/src/airflow_controller.rs

```diff
@@ -306,6 +293,26 @@ pub async fn reconcile_airflow(airflow: Arc<AirflowCluster>, ctx: Arc<Ctx>) -> R
     )
     .context(CreateClusterResourcesSnafu)?;
 
+    let (rbac_sa, rbac_rolebinding) = build_rbac_resources(
+        airflow.as_ref(),
+        "airflow",
+        cluster_resources.get_required_labels(),
+    )
+    .context(BuildRBACObjectsSnafu)?;
+
+    cluster_resources
+        .add(client, rbac_sa.clone())
+        .await
+        .with_context(|_| ApplyServiceAccountSnafu {
+            name: rbac_sa.name_unchecked(),
+        })?;
+    cluster_resources
+        .add(client, rbac_rolebinding.clone())
+        .await
+        .with_context(|_| ApplyRoleBindingSnafu {
+            name: rbac_rolebinding.name_unchecked(),
+        })?;
```

Comment:
You can remove the name from the error (not used in the display string and will be propagated anyways).
This avoids the clones as well.
```suggestion
    let rbac_sa = cluster_resources
        .add(client, rbac_sa)
        .await
        .context(ApplyServiceAccountSnafu)?;
    cluster_resources
        .add(client, rbac_rolebinding)
        .await
        .context(ApplyRoleBindingSnafu)?;
```